src/components/NavBar/OidcLoginComponent.jsx

```diff
@@ -37,16 +36,18 @@ const OidcLoginComponent = () => {
       <Box sx={{ marginRight: '32px' }} />
       <Button
         variant="contained"
-        type="submit"
+        type="button"
         color="secondary"
         size="large"
         aria-label="Login Button"
         onClick={() => {
           loginHandler();
+          localStorage.setItem('oidcIssuer', oidcIssuer);
         }}
         onKeyUp={(event) => {
```

Comment:
`onKeyUp` doesn't seem to work even after pressing the `Enter` key even before switching the button type from "submit" to "button". Think the event is not being caught by the component. Could address this in another PR.